File: kotlin.Ver/app/src/main/java/com/exercise/ckaiself/kaiyan_kotlinver/mvp/model/bean/Issue.kt
package com.exercise.ckaiself.kaiyan_kotlinver.mvp.model.bean

import java.io.Serializable


/**
 * @author Ckai
 * @date 2019/1/15 11:19 AM
 * @description
 */

data class Issue(
    val itemList: ArrayList<Item>,
    var count: Int,
    val total: Int,
    val nextPageUrl: String,
    val adExist: Boolean
)

File: flutterversion/lib/bean/Issue.dart
import '../bean/Item.dart';
import 'package:json_annotation/json_annotation.dart';

part 'Issue.g.dart';

@JsonSerializable()
class Issue {
  List<Item> itemList;
  num count;
  num total;
  String nextPageUrl;
  bool adExist;

  Issue(
      {this.itemList, this.count, this.total, this.nextPageUrl, this.adExist});

  factory Issue.fromJson(Map<String, dynamic> json) => _$IssueFromJson(json);
}
